content/en/graphing/metrics/distributions.md

```diff
@@ -15,19 +15,19 @@ This feature is in beta. <a href="https://docs.datadoghq.com/help/">Contact Data
 
 ## Overview
 
-Distributions are a metric type in Agent v6 that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure server-side.  This can be thought of as a global version of the [Histogram metric][1], which measures at the Agent-level the statistical distribution of values on a single host.
+Distributions are a metric type in that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure server-side.
 
-Global distributions are designed to instrument logical objects, like services, independently from the underlying hosts. Unlike histograms which aggregate on the Agent-side, global distributions send all raw data collected during the flush interval and the aggregation occurs server-side. Because the underlying data structure has not been aggregated and represents raw data, distributions provide two major features:
+Global distributions are designed to instrument logical objects, like services, independently from the underlying hosts. Unlike histograms [1] which aggregate on the Agent-side, global distributions send all raw data collected during the flush interval and the aggregation occurs server-side. Because the underlying data structure has not been aggregated and represents raw data, distributions provide two major features:
 
 * **Calculation of percentile aggregations**: Percentile aggregations (p50, p75, p90, p95, p99) are calculated from the raw data across all hosts, and are therefore globally accurate.
 
 * **Customization of tagging**: This functionality allows you to control the tagging scheme for metrics for which host-level granularity is not necessary (e.g. transactions per second for a checkout service).
 
-See the [Developer Tools section][2] for more implementation details.
+See the [Developer Tools section][2] for more implementation details. Please note that since distributions are a new metric type, they should be instrumented under new metric names during submission to Datadog.
```

Comment:
```suggestion
See the [Developer Tools section][2] for more implementation details. **Note**: Since distributions are a different metric type, any previously submitted metrics should be instrumented under new metric names.
```

Reply:
same as earlier comment - distributions should be submitted under new metric names not that previously submitted metrics should be renamed